examples/io-csv-datamapping/io_csv.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/io;
+
+
+// Define the record to bind the data with
+type Employee record{
+    int id;
+    string name;
+    int salary;
+};
+
+public function main() returns error? {
+    // Initializes the CSV file path and content.
+    string csvFilePath1 = "./files/csvFile1.csv";
+    string csvFilePath2 = "./files/csvFile2.csv";
+    Employee[] csvContent = [{id:1, name:"James", salary:10000}, {id:2, name:"Nathan", salary:150000},
+    {id:3, name:"Ronald", salary:120000}, {id:4, name:"Roy", salary:6000},
+    {id:5, name:"Oliver", salary:1100000}];
+
+    // Writes the given content record[] to a CSV file.
+    check io:fileWriteCsv(csvFilePath1, csvContent);
+    // If the write operation was successful, then, performs a read operation
+    // to read the CSV content as a array of records.
```

Comment:
We can have same comments, we added to other csv BBEs

Reply:
Updated in [a00284d](https://github.com/ballerina-platform/ballerina-distribution/pull/3038/commits/a00284d7cea7f916b48bee1cfbaddf94ceb0c0d0)